pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/file/FileProcessor.java

```diff
@@ -44,13 +45,15 @@ public abstract class FileProcessor {
     private final EventWriter<byte[]> writer;
     private final TransactionCoordinator transactionCoordinator;
     private final EventGenerator eventGenerator;
+    private final Path movedFilesDirectory;
```

Comment:
completedFilesDirectory is a better name.

Reply:
This directory is parent of completed, failed and db file, that's why kept as just moved files, it might be failed or completed.